app/controllers/application_controller.rb

```diff
@@ -13,4 +13,13 @@ def authenticate_admin
       errors: ['purmission denied.']
     }, status: 401
   end
+
+  def authenticate_admin_and_login_user
```

Comment:
😢

Reply:
LGTM
[![LGTM](https://www.lgtm.in/p/kO2icCXHn)](https://www.lgtm.in/i/kO2icCXHn)